src/mantine-hooks/src/use-validated-state/use-validated-state.test.ts

```diff
@@ -0,0 +1,29 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useValidatedState } from './use-validated-state';
+
+describe('@mantine/hooks/use-validated-state', () => {
+  it('returns inital value', () => {
+    const hookValid = renderHook(() => useValidatedState('test', (val) => val === 'test'));
+    expect(hookValid.result.current[0].lastValidValue).toBe('test');
+    expect(hookValid.result.current[0].valid).toBe(true);
+    expect(hookValid.result.current[0].value).toBe('test');
+    expect(typeof hookValid.result.current[1]).toBe('function');
+
+    const hookUnvalid = renderHook(() => useValidatedState('test', (val) => val === 'tests'));
```

Comment:
Unvalid -> Invalid, you can install https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker to find spelling mistakes